compiler/parser/parser.peg

```diff
@@ -725,7 +725,10 @@ Cast
     }
 
 Function
-  = !FuncGuard fn:IdentifierName __ "(" __ args:OptionalExprs __ ")" {
+  = !FuncGuard fn:IdentifierName __ "(" __ args:OptionalExprs __ ")" where:WhereClause {
+      RETURN(MAP("kind": "Call", "name": fn, "args": args, "where": where))
+    }
+  / !FuncGuard fn:IdentifierName __ "(" __ args:OptionalExprs __ ")" {
```

Comment:
I think we should be consistent with the way we've handled this elsewhere, which would be to add
```
  = !FuncGuard fn:IdentifierName __ "(" __ args:OptionalExprs __ ")" where:WhereClause? {
      RETURN(MAP("kind": "Call", "name": fn, "args": args, "where": where))
    }
```
here and then add `"where": NULL` to the other `"kind": "Call"` maps.

Reply:
Ok!

Reply:
Back to the way it was!